playbooks/portals-docker-command.yml

```diff
@@ -0,0 +1,55 @@
+- name: Run Docker Command on Skynet Webportal
+  hosts: webportals
+  remote_user: "{{ webportal_user }}"
+  gather_facts: False
+
+  # Target 3 servers at a time.
+  serial: 3
+
+  # Stop on first error, do not execute on the next host
+  any_errors_fatal: True
+
+  # Playbook specific vars
+  vars:
+    max_hosts: "{{ groups['webportals'] | length - 1 }}"
+
+  tasks:
+    # Check '--limit' is used. This is to protect against taking the entire
+    # fleet down
+    - name: Fail if you are targeting all dev and prod webportals
+      include_tasks: tasks/host-limit-check.yml
+
+    # Skip portals on takedown as they are already out of the LB
+    - block:
+        - name: Stop play for portals in webportals_takedown group
+          debug:
+            msg: "Will not execute command on portals in webportals_takedown group"
+
+        - name: Stop play for portals in webportals_takedown group
+          meta: end_host
+
+      when: "'webportals_takedown' in group_names"
+
+    # Check/install Ansible prerequisites
+    - name: Include preparing portal prerequisites
+      include_tasks: tasks/portals-prepare.yml
+
+    # Disable health check
+    - name: Include disabling portal health check
+      include_tasks: tasks/portal-health-check-disable.yml
```

Comment:
This could be optional. The reason for that is that sometimes we will want to do something trivial like change a config value on all servers that won't take effect until the server is restarted. We don't need to add extra stress to the network by disabling servers.

Reply:
I disagree. That adds a human decision into the process which isn't scalable. 
We should handle all events the same way regardless of size. Something might seem trivial at the time but have an unintended consequence.